__tests__/app.test.js

```diff
@@ -282,4 +281,25 @@ describe('app testing', () => {
             })
         })
     })
+    describe('get endpoints', () => {
+        test('Returns with all the endpoints available to the api', () => {
+            return request(app)
+            .get('/api')
+            .expect(200)
+            .then(({ body: {endpoints} }) => {
+                const parsedEndpoints = JSON.parse(endpoints);
+                expect(parsedEndpoints).toEqual(
+                    expect.objectContaining({
+                        "GET /api": expect.any(Object),
+                        "GET /api/articles": expect.any(Object),
+                        "GET /api/articles/:article_id": expect.any(Object),
+                        "GET /api/articles/:article_id/comments": expect.any(Object),
+                        "POST /api/articles/:article_id/comments": expect.any(Object),
+                        "PATCH /api/articles/:article_id": expect.any(Object),
+                        "GET /api/users": expect.any(Object)
+                    })
```

Comment:
Some of these are arrays, so will automatically come back as a false positive as they are technically objects. Can you be more specific to the ones that you actually expect an array on?

Reply:
^^ This is wrong ❌  -> Can approve.